lib/pal-platform/pal-platform.json

```diff
@@ -228,11 +228,11 @@
         "to": "Middleware/trusted_storage/trusted_storage"
       },
       "parsec_se_driver": {
-        "version": "0.5.0",
+        "version": "0.6.0",
         "from": {
           "protocol": "git",
           "location": "https://github.com/parallaxsecond/parsec-se-driver.git",
-          "tag": "0.5.0"
+          "tag": "0.6.0"
```

Comment:
We probably need to move upto 0.8.1, actually. @petedyerarm  @jenia81 @moshe-shahar

Reply:
0.6.0 is the latest version of the SE driver. 
0.8.1 is the version of the parsec-service.
So this is correct.